zzre/game/systems/ui/ScrBookMenu.cs

```diff
@@ -27,8 +50,12 @@ protected override void HandleOpen(in messages.ui.OpenBookMenu message)
 
         var entity = World.CreateEntity();
         entity.Set<components.ui.ScrBookMenu>();
-        ref var bookMenu = ref entity.Get<components.ui.ScrBookMenu>();
-        bookMenu.Inventory = inventory;
+        ref var book = ref entity.Get<components.ui.ScrBookMenu>();
+        book.Inventory = inventory;
+        book.Fairies = db.Fairies.OrderBy(fairyRow => fairyRow.CardId.EntityId).ToArray();
+        book.FairyButtons = new Dictionary<components.ui.ElementId, FairyRow>();
+        book.Sidebar = World.CreateEntity();
+        book.Crosshair = World.CreateEntity();
```

Comment:
Please leave these two default and add a check before disposing. Currently when the screen is closed without showing any fairy details these would not be disposed and be kept alive until restart.